openpype/modules/deadline/plugins/publish/submit_3dmax_deadline.py

```diff
@@ -0,0 +1,237 @@
+import os
+import json
+import getpass
+
+import requests
+import pyblish.api
+
+from openpype.pipeline import legacy_io
+from openpype.settings import get_project_settings
+from openpype.hosts.max.api.lib import (
+    get_current_renderer,
+    get_multipass_setting
+)
+from openpype.hosts.max.api.lib_rendersettings import RenderSettings
+
+
+class MaxSubmitRenderDeadline(pyblish.api.InstancePlugin):
```

Comment:
I think this should use `AbstractSubmitDeadline` - `from openpype_modules.deadline import abstract_submit_deadline`.

Because that is handling all deadline job options and its serialization. It should be unified across all Deadline supporting hosts. Almost all (except Houdini and Nuke) are using this.